server/plugin.go

```diff
@@ -94,14 +86,13 @@ func (p *Plugin) OnActivate() error {
 
 	p.store = store.NewService(p.mm, p.conf)
 	// manifest store
-	conf := p.conf.GetConfig()
 	mstore := p.store.Manifest
 	mstore.Configure(conf)
-	// TODO: uses the default bucket name, do we need it customizeable?
-	manifestBucket := apps.S3BucketNameWithDefaults("")
-	err = mstore.InitGlobal(p.aws, manifestBucket, p.httpOut)
-	if err != nil {
-		return errors.Wrap(err, "failed to initialize the global manifest list from marketplace")
+	if conf.MattermostCloudMode {
```

Comment:
How would on-prem customers who use AWS to deploy there Apps listed?

cc @DHaussermann as we talked about this offline in the team meeting

Reply:
I have been assuming that `appsctl` would provide the functionality and the customer would use it with the appropriate credentials to manage their deployment.